Source/drlg_l2.cpp

```diff
@@ -1766,7 +1766,7 @@ static void DRLG_L2Subs()
 
 	for (y = 0; y < DMAXY; y++) {
 		for (x = 0; x < DMAXX; x++) {
-			if ((x < nSx1 || x > nSx2) && (y < nSy1 || y > nSy2) && random_(0, 4) == 0) {
+			if ((x < nSx1 || x > nSx2) && (y < nSy1 || y > nSy2) && random_(0, 4) == 0) { // BUGFIX: Should be (x < nSx1 || x > nSx2 || y < nSy1 || y >= nSy2)

```

Comment:
Should the last `>=` be `>` instead? Typo?

Reply:
I'm not 100% sure, it's also like that in another location and I haven't determined if the second coordinate is actually on the inside of the room rather then marking the edge.